index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <main>
+        <!-- The Header which includes the logo and hamburger menu -->
+        <header>
+            <a class="logo">My Logo</a>
+            <nav>
+                <i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+            </nav>
+        </header>
+        <!-- The Headline section that contains a description of the author -->
```

Comment:
[OPTIONAL] Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
**Kindly remove all commented code in your project.**